docs/user-guide/pods/index.md

```diff
@@ -40,22 +40,6 @@ In terms of [Docker](https://www.docker.com/) constructs, a pod is modelled as
 a group of Docker containers with shared namespaces and shared
 [volumes](/docs/user-guide/volumes/). PID namespace sharing is not yet implemented in Docker.
```

Comment:
PID namespaces are actually supported by Docker now. 
https://github.com/kubernetes/kubernetes/issues/1615#issuecomment-220150807

I'd just remove that sentence.